py/occampy2/wrappers/sbm_manager.py

```diff
@@ -0,0 +1,42 @@
+import occam
+from .variable_list import VariableList
+
+
+class SBMManager:
+    """
+    Wrapper class for SBMManager
+    """
+
+    def __init__(self, ref=None):
+        """
+        :param: ref: the reference to the SBMManager object returned from the CPP engine
+        """
+        # Create new reference if one not given
+        self._ref = ref or occam.SBMManager()
+        self._model = None
+
+    @property
+    def variable_list(self):
+        return VariableList(self._ref.getVariableList())

```

Comment:
As many of these methods are the same as `VBMManager`, a class `Manager` should be created which has all the common functionality and both `VBMManager` and `SBMManager` should inherit from it

Reply:
Good idea, I can throw that in as a stub for now.